mne/preprocessing/nirs/_tddr.py

```diff
@@ -132,6 +132,8 @@ def _TDDR(signal, sample_rate):
         sigma = 1.4826 * np.median(dev)
 
         # Step 3d. Scale deviations by standard deviation and tuning parameter
+        if sigma == 0:
```

Comment:
Does this also indicate a flat channel? As above.

Reply:
Do try and keep the code as close to the original as possible could we add the flat signal check as a block at the front?

Reply:
We could but I don't mind the small diff that's a clear fix here. It ends up being less clean to do it at the beginning I think because it creates redundant calculations